docs/content/common-subs/install-helm-kind.md

```diff
@@ -0,0 +1,14 @@
+<!--install-helm-kind-start-->
+```shell hl_lines="6"
+KUBECONFIG=~/.kube/config kubectl config use-context ks-core  
+kubectl create namespace kubestellar  
+
+helm repo add kubestellar https://helm.kubestellar.io
+helm repo update
+helm install kubestellar/kubestellar-core \
```

Comment:
The `helm` command also consumes the choice of kubeconfig file, in the regular ways: by a setting of the KUBECONFIG envar or with the `--kubeconfig` command line flag.